src/index.html

```diff
@@ -17,16 +17,29 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+      class="search__bar"
+    >
+      <input
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+        class="search__bar--input input__big input__icon icon__big"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class names 'input__big', 'input__icon', 'icon__big' seem to describe styles rather than content. Consider using more descriptive class names that reflect the function or content of the element.